conans/client/downloaders/file_downloader.py

```diff
@@ -90,24 +90,6 @@ def _download_file(self, url, auth, headers, file_path, verify_ssl, try_resume=F
                 raise AuthenticationException()
             raise ConanException("Error %d downloading file %s" % (response.status_code, url))
 
-        def read_response(chunk_size, path=None):
-            ret = None
-            downloaded_size = range_start
-            if path:
-                mkdir(os.path.dirname(path))
-                mode = "ab" if range_start else "wb"
-                with open(path, mode) as file_handler:
-                    for chunk in response.iter_content(chunk_size):
-                        file_handler.write(chunk)
-                        downloaded_size += len(chunk)
-            else:
-                ret_data = bytearray()
```

Comment:
This code was never happening, as it was used in 1.X to download files from server without saving to disk, something that doesn't happen anymore in 2.0